website/src/pages/dictionary/index.js

```diff
@@ -379,7 +379,7 @@ function DataDictionary() {
                 />
               </Display>
 
-              <Display visible={selectedTab === TAB_STATE.OVERVIEW}>
```

Comment:
this can technically be left as is eh? the tabs are hidden so nothing can change the state :P

Reply:
this is true - wanted to be extra clear because this file is slowly filling up with "hidden features" :p